lib/app/routes/core.rb

```diff
@@ -114,10 +114,6 @@ def dashboard_assignment_nav(language, slug=nil, counts=nil)
         </li>}
         end
 
-        def show_pending_submissions?(language)
```

Comment:
How is the removal of this method related to this change to allow unauthenticated users to view a submission?